BBKoffieTuin/Assets/Scripts/PhoneDirection.cs

```diff
@@ -0,0 +1,49 @@
+using System;
+using TMPro;
+using UnityEngine;
+using UnityEngine.Events;
+
+public class PhoneDirection : MonoBehaviour
+{
+    [SerializeField] private float rotationTolerance = 1f;
+    
+    private Vector2 _currentPos;
+    private Vector2 _nextPos = new Vector2(4.856711f, 52.390652f);
+
+    private float _lastNorthRotation = 0;
+    private float _lastTargetPointRotation = 0;
+
+    public UnityEvent<float> onNortherPointChange = new UnityEvent<float>();
+    public UnityEvent<float> onTargetPointChange = new UnityEvent<float>();
+
+    private void Awake()
+    {
+        Input.compass.enabled = true;
+    }
+
+    void Update()
+    {
+        float latitude = Input.location.lastData.latitude;
+        float longitude = Input.location.lastData.longitude;
```

Comment:
Boven aan de input GpsServices.Instance.gpsServiceStarted anders krijg je warning wanneer er gps niet aan is